package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "web3-react-native-dapp-wagmi",
+  "version": "1.0.0",
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web"
```

Comment:
Drop web as we do not support his right?

Reply:
Done